source/sass/mixins/_typography.scss

```diff
@@ -112,6 +112,11 @@
   }
 }
 
+@mixin figure-caption-typography() {
```

Comment:
This name is very specific, makes sense to include here only if it's more generic (ie can be used in places other than figure captions).